postgres/postgresql.conf.erb

```diff
@@ -33,13 +32,20 @@ lc_numeric = 'en_AU.UTF-8'
 lc_time = 'en_AU.UTF-8'
 default_text_search_config = 'pg_catalog.english'
 
+idle_in_transaction_session_timeout = '120s'
+
 # These settings only apply to the master.
-wal_level = hot_standby
+wal_level = replica
```

Comment:
It's worth reading the [limitations and restrictions](https://github.com/2ndQuadrant/pglogical#limitations-and-restrictions). On slight downside is that we have to enable replication for each database in the master.